zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Query.scala

```diff
@@ -17,66 +17,66 @@ trait Statement {}
   * Augments the source data to contain more fields.
   *
   * @param sourceKeys
-  * @param lookupDataset
+  * @param dataset
   * @param lookupKeys
   * @param selectValues
   * @param as
   */
-class LookupStatement(val sourceKeys: Seq[String],
-                      val lookupDataset: String,
-                      val lookupKeys: Seq[String],
-                      val selectValues: Seq[String],
-                      val as: Seq[String]
-                     ) extends Statement {
+case class LookupStatement(val sourceKeys: Seq[String],
```

Comment:
`case class` is a syntax sugar for `struct data`. Here is the [doc](http://docs.scala-lang.org/tutorials/tour/case-classes.html)